### DISCUSSION TITLE
Python as a plugin

### DISCUSSION THREAD

**Discussion by techwithanant** [jhuckaby/Cronicle#521] [Q&A] [answered]
Hi,

I am very new to this and trying to create a python plugin which would except some parameters. 
I created a plugin with below execution -

/usr/bin/python3 /usr/local/bin/my_python_file.py

The content of my_python_file.py is - 

print(“hello world”)

When I schedule this plugin and run, I do get an output in logs but also the job fails with error “job exited without giving any status”

Is there anything I am missing? Thanks in advance.

**Comment by techwithanant** [comment]
After some more digging I found this repo https://github.com/josephdadams/CroniclePlugins

Probably I need to create my plugins like this

**Comment by dheise-rheinenergie-trading-com** [comment, marked as answer]
You have to return an exit code in a json structure. Have a look into the examples from Cronicle: https://github.com/jhuckaby/Cronicle#sample-node-plugin

My custom python plugins exits with this lines:

```
...
print('{"complete": 1, "code": 0, "description": "successfully executed"}')
sys.exit(0)
```

**Reply by techwithanant** [reply]
It worked. Thanks a lot. Really appreciate it :)
